types/culori/test/mapper.test.ts

```diff
@@ -46,3 +46,11 @@ mapper(
     undefined,
     true,
 );
+
+const halver = mapper(v => v / 2, null);
```

Comment:
May I ask what the behavior was before the changes you are proposing in this PR?